demos/bert_question_answering_demo/python/bert_question_answering_demo.py

```diff
@@ -24,7 +24,7 @@
 import numpy as np
```

Comment:
I think this would be a good time to remove "bert" from the name of the demo, since our general convention is to name demos after tasks rather than models.

Reply:
I agree, but we should do it in the separate PR.